test/TestOverrides.cpp

```diff
@@ -0,0 +1,85 @@
+#include <jni.h>
+#include "API.h"
+#include "Test.h"
+#include <string.h>
+#include <assert.h>
+
+static jobject s_CustomClassLoader = 0;
+static jmethodID s_ClassForNameMethod = 0;
+static int s_ClassLoaderCallCount = 0;
+
+struct JNIOverrides
+{
+    typedef jclass(*FindClass)(JNIEnv*, const char*);
+
+    FindClass findClass;
+};

```

Comment:
This is ugly. Why are you duping struct instead of just using the one you defined in header?

Reply:
Is this even used? I might be blind, but it looks like a leftover.

Reply:
not used